statistics/2025-04-28-p2ra/index.qmd

```diff
@@ -0,0 +1,1111 @@
+
+```{r}
+posteriors %>%
+  ggplot(aes(phi_ww, color=group)) +
+  geom_density() +
+  prior_density(10^seq(-3, 3, length.out=100), prior_phi_ww, log10x = TRUE) +
+  scale_x_log10(limits = c(1e-3, 1e3)) +
+  facet_wrap(vars(species), ncol = 4) +
+  labs(y = "") +
+  theme(legend.position = "", axis.text.y = element_blank())
+```
+
+The wastewater data has more non-zero counts and thus more data to infer dispersion.
+I'm not sure I trust any of these though.
+
+Generally, the species that have high predicted overdispersion relative to the prior (low $\phi$) are ones with fewer positive wastewater samples. 
+That could be because there are more zero-read samples than expected given the counts in the non-zero-read samples.
+
+Something funny is going on with HPIV1, HPIV2, and HMPV-1 which are unobserved in the swabs and only observed in 1-3 the wastewater samples.
+It's probably worth following up on these, unless we want to throw out the unobserved swab viruses altogether.
+
+Another option would be to use a combined model where all the viruses share (totally or hierarchically) $\phi$ parameters. This could make sense since lots of the sources of overdispersion are probably shared between viruses.
+If true, this would allow us to get better estimates of $\phi$.
+
+## Joint Posterior Distributions
+
+We can also examine relationships between different parameters in the posterior.
+In the following plots, each point is a sample from the joint posterior of two parameters.
+The curves are an estimate of the conditional mean of the y-axis parameter given the x-axis parameter.
+
+### Realtionships with prevalence
+
+Prevalence vs $\mu_1$ (note log x-scale not logit):
+
+```{r}
+posteriors %>%
+  ggplot(aes(mu_1, p)) +
+  geom_point(alpha = 0.05, size = 0.1) +
+  geom_smooth() +
+  scale_x_log10(limits = c(1e-7, 1), labels = scientific) +
+  scale_y_log10(limits = c(1e-6, 1), labels = scientific) +
+  facet_wrap(~species, ncol=6)
+```
+
+For small $\mu$, the prevalence must be large to account for detection.
+For larger $\mu$, this relationship declines because observing a virus in a swab puts a lower bound on the possible prevalences.
+(I'm not sure why you also see this pattern in the unobserved viruses, though.)
+
+
+```{r}
+posteriors %>%
+  ggplot(aes(phi_1, p)) +
+  geom_point(alpha = 0.05, size = 0.1) +
+  geom_smooth() +
+  scale_x_log10(limits = c(1e-2, 10), labels=scientific) +
+  scale_y_log10(limits = c(1e-6, 1), labels = scientific) +
+  facet_wrap(~species, ncol = 6)
+```
+
+There appears to be a relationship between the swab inverse overdispersion and the prevalence estimates, suggesting that if we used a prior that allowed smaller $phi_1$, we might get larger prevalence estimates for some viruses.
+
+This makes sense because if we only see a virus a few times and overdispersion is very large, the true prevalence might be lower than we think and our observations are lucky outliers.
```

Comment:
But if overdispersion is very large, couldn't true prevalence also be higher than we think, and our observations are lucky outliers the other way?